### DISCUSSION TITLE
Xiaomi Smart Doorbell 3

### DISCUSSION THREAD

**Discussion by dimkagithub** [merdok/homebridge-miot#510] [General]
Hi!
I am going to buy this bell (madv.cateye.mowl3g (Xiaomi Smart Doorbell 3)), but there are questions.
Can you tell me what functions will work for this bell?
Will the image be transmitted to the iPhone?
Will the alarm sound on the phone when someone rings the bell?
Thanks.

**Comment by merdok** [comment]
Although the device has quite some features based on the miot spec http://www.merdok.org/miotspec/?model=madv.cateye.mowl3g you will only be able to configure the device. Neither a notification nor a video will be transmitted to your device as this functionality is currently not implemented.
